consensus/obcpbft/obc-batch.go

```diff
@@ -408,6 +418,11 @@ func (op *obcBatch) resubmitStaleRequest(c custodyInfo) {
 
 // allow the primary to send a batch when the timer expires
 func (op *obcBatch) main() {
+
```

Comment:
if `op.isSufficientlyConnected` is closed, this reduces to an unconditional `<- op.isSufficientlyConnected`